Dockerfile

```diff
@@ -0,0 +1,100 @@
+#{{{ demo_lti1p3
+FROM php:7.4-fpm-alpine as demo_lti1p3-base
+ENV MAKEFLAGS "-j8"
+
+RUN set -eux; \
+        cd /; \
+        apk add --no-cache --virtual .build-deps \
+            autoconf \
+            g++ \
+            unzip \
+            icu-dev \
+            sudo \
+            libmcrypt-dev \
+            libpng-dev \
+            librdkafka-dev \
+            libxml2-dev \
+            linux-headers \
+            make \
+            oniguruma-dev \
+            postgresql-dev \
+            procps \
+            unzip \
+            wget \
+            zip \
+            libzip-dev \
+            zlib-dev \
+            zstd-dev \
+        ; \
+        yes "" | pecl install redis igbinary \
+        ;\
+        docker-php-ext-install intl; \
+        docker-php-ext-install gd; \
+        docker-php-ext-install opcache; \
+        docker-php-ext-install zip; \
+        docker-php-ext-install calendar; \
+        docker-php-ext-install sockets \
+        ; \
+        docker-php-ext-enable redis; \
+        docker-php-ext-enable igbinary; \
+        runDeps="$( \
+                scanelf --needed --nobanner --format '%n#p' --recursive /usr/local \
+                | tr ',' '\n' \
+                | sort -u \
+                | awk 'system("[ -e /usr/local/lib/" $1 " ]") == 0 { next } { print "so:" $1 }' \
+        )"; \
+        apk add --no-cache $runDeps; \
+        apk del --no-network .build-deps; \
+        rm -rf /var/www/html; \
+        chmod 0777 /tmp/;
+
+RUN set -eux; \
+        { \
+            echo 'upload_max_filesize=60M'; \
+            echo 'post_max_filesize=60M'; \
+            echo 'post_max_size=60M'; \
+        } >> /usr/local/etc/php/conf.d/deliver.ini \
+        ;
+
+WORKDIR /var/www/html
+#}}} demo_lti1p3-base
+
+#{{{ composer
+FROM demo_lti1p3-base AS composer
+
+ARG SSH_KEY
+ENV SSH_KEY=$SSH_KEY
```

Comment:
all repos used in this app are public , no need to have SSH keys for composer

Reply:
removed